exporter/sumologicexporter/fields.go

```diff
@@ -33,3 +33,12 @@ func (f fields) string() string {
 
 	return strings.Join(rv, ", ")
 }
+
+func (f fields) asInterfaceMap() map[string]interface{} {
+	rv := make(map[string]interface{}, len(f))
```

Comment:
I prefer more meaningful names.

Reply:
It all depends on the context I'd say. In this particular `func` having just 5 lines of code and a single loop copying values I'm not sure if there is a name that would increase the readability.

There are many blogs/presentations on this subject, even by Russ Cox :) 
* https://research.swtch.com/names
* https://talks.golang.org/2014/names.slide

Reply:
`rv` stands for `returnValue` in this case and this funcion should go away with #15 anyway